var/spack/repos/builtin/packages/braker/package.py

```diff
@@ -31,6 +31,8 @@ class Braker(Package):
 
     homepage = "http://exon.gatech.edu/braker1.html"
     url      = "http://bioinf.uni-greifswald.de/augustus/binaries/BRAKER1_v1.11.tar.gz"
+    list_url = "http://bioinf.uni-greifswald.de/augustus/binaries/"
```

Comment:
If you add `old` to the end of `list_url` and drop the `list_depth` it will result in significantly fewer searches. By default, Spack will check the `url` above. If it doesn't find something, it will check in `list_url`.